package.json

```diff
@@ -31,7 +31,12 @@
     "node": "8 - 9"
   },
   "dependencies": {
-    "node-pre-gyp": "^0.9.1"
+    "del": "^3.0.0",
+    "glob": "^7.1.3",
+    "glob-promise": "^3.4.0",
+    "nan": "^2.10.0",
+    "node-pre-gyp": "^0.9.1",
+    "underscore": "^1.9.1"
```

Comment:
is it necessary? or it's used only in tests?

Reply:
glob & underscore are used in tests only.